tests/test_display.py

```diff
@@ -69,13 +62,12 @@ def test_astropy_helpers(testdir, capsys):
         del builtins._ASTROPY_SETUP_
```

Comment:
I think _ASTROPY_SETUP_ is gone as well :)

Reply:
Actually, should I just removed the whole test function? Sorry for the premature merge. :/

Reply:
See #30